plugins/woocommerce-admin/client/launch-your-store/settings/slotfill.js

```diff
@@ -11,13 +11,15 @@ import {
 	useState,
 	createInterpolateElement,
 	createElement,
+	useEffect,
 } from '@wordpress/element';
 import { registerPlugin } from '@wordpress/plugins';
 import { __ } from '@wordpress/i18n';
 import classNames from 'classnames';
 import { useCopyToClipboard } from '@wordpress/compose';
 import { recordEvent } from '@woocommerce/tracks';
 import { getSetting } from '@woocommerce/settings';
+import { isEqual } from 'lodash';
```

Comment:
We can replace `isEqual` with [fastDeepEqual](https://href.li/?https://www.npmjs.com/package/fast-deep-equal) so we don't have to add more lodash functions to our codebase (pdToLP-yS-p2).

Reply:
@chihsuan TIL! Replaced it with simple === instead in https://github.com/woocommerce/woocommerce/pull/47316/commits/6e72292f72acc68fb1e1ea1a007742c873b0a463